src/main/java/jetbrains/buildServer/iaa/AutoAssignerBuildFeature.java

```diff
@@ -51,6 +52,25 @@ public String getEditParametersUrl() {
     return myEditUrl;
   }
 
+  @NotNull
+  @Override
+  public String describeParameters(@NotNull Map<String, String> params) {
+    final String userName = params.get(Constants.DEFAULT_RESPONSIBLE);
+    final String usersToIgnore = params.get(Constants.BLACK_LIST);
+    StringBuilder sb = new StringBuilder();
+    if (userName != null && !userName.isEmpty()){
+      sb.append("Default responsible: [").append(userName).append("]").append("\n");
+    } else {
+      sb.append("Default responsible is not specified").append("\n");
+    }
+    if (usersToIgnore != null && !usersToIgnore.isEmpty()){
+      sb.append("Users to ignore: [").append(usersToIgnore).append("]");
+    } else {
+      sb.append("Users to ignore are not specified");
```

Comment:
It looks like we should describe only configured fields, so notes "Users to ignore are not specified" / "Default responsible is not specified" may not be appended there